### ISSUE TITLE
Various prometheus rules/alerts are not scoped to the enmasse namespace

### ISSUE THREAD

**Issue by david-martin** [EnMasseProject/enmasse#2897]
**Describe the bug**
The following alerts don't have a namespace label scope
https://github.com/EnMasseProject/enmasse/blob/4fe9760b9cb78c977b43b7a68dede7af4b4b9f03/templates/prometheus-rules/010-PrometheusRules-prometheus.yaml#L44-L65
This causes alerts to be triggered for other namespaces in the cluster (see screenshot for example)

kube_state_metrics are already being pulled into the prometheus instance I'm using, which is scoped across a number of namespaces. That would explain why the alerts are finding the other metrics. However, I'd like if these rules could be scoped to the enmasse namespace instead of having to limit what I'm pulling in from kube_state_metrics

**To Reproduce**
1. Install enmasse
2. Install the monitoring stack, including kube_state_metrics (with permissions to gather metrics from more than just the enmasse namespace)

**Expected behavior**
The EnMasse alerts only trigger if there are problems in the EnMasse namespace

**Screenshots**
![image](https://user-images.githubusercontent.com/878251/59349016-8b71e080-8d10-11e9-9d92-94fcd3fb490f.png)

**Additional context**
This is an integreatly cluster where there's a single monitoring stack reconciling monitoring CRs from multiple services

**Comment by lulf** [comment]
I think this is fixed in 1.4 where rules are on this form:

```
     - alert: TerminatingPods
      annotations:
        description: Pod {{ "{{" }} $labels.pod {{ "}}" }} has been in a terminating state for over 5 minutes
        severity: warning
      expr: kube_pod_container_status_terminated{exported_namespace="{{ index .Params "Namespace" }}"} == 1
      for: 180s
```

Let me know if this is the expected form, @david-martin

**Comment by david-martin** [comment]
> Let me know if this is the expected form

@lulf Yes, that should do the trick
